### ISSUE TITLE
INSERT ... ON DUPLICATE KEY

### ISSUE THREAD

**Issue by lem9** [phpmyadmin/phpmyadmin#11628]
Syntax flagged as error by the parser. See http://dev.mysql.com/doc/refman/5.7/en/insert-on-duplicate.html and http://stackoverflow.com/questions/33407551/on-duplicate-key-unrecognized-in-phpmyadmin.

**Comment by nijel** [comment]
Confirmed with current parser:

```
#1: Unrecognized keyword. (near "KEY" at position 69)
#2: Unexpected token. (near "=" at position 88)
#3: This type of clause was previously parsed. (near "VALUES" at position 89)
#4: Unrecognized statement type. (near "VALUES" at position 89)
```

**Comment by techpulse** [comment]
http://i.imgur.com/aeJatX2.png

**Comment by nijel** [comment]
I've released SQL parser 3.4.5 and included it in QA_4_6 branch.
